internal/common/logging/stacktrace.go

```diff
@@ -5,18 +5,34 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+const Stacktrace = "Stacktrace"
```

Comment:
```suggestion
const Stacktrace = "stacktrace"
```
I prefer lower-case.